Explorer/Assets/DCL/CommunicationData/URLHelpers/URN.cs

```diff
@@ -116,23 +108,23 @@ void LogError()
         }
 
         public override int GetHashCode() =>
-            urn != null ? StringComparer.OrdinalIgnoreCase.GetHashCode(urn) : 0;
+            lowercaseUrn != null ? StringComparer.OrdinalIgnoreCase.GetHashCode(lowercaseUrn) : 0;
 
         public URN Shorten()
         {
-            if (string.IsNullOrEmpty(urn)) return urn;
+            if (string.IsNullOrEmpty(originalUrn)) return originalUrn;
             // Third party collections do not include the tokenId and have 7 parts, so we must keep all of them
-            if (IsThirdPartyCollection()) return urn;
+            if (IsThirdPartyCollection()) return originalUrn;
 
             int index = -1;
 
             for (var i = 0; i < SHORTEN_URN_PARTS; i++)
             {
-                index = urn.IndexOf(':', index + 1);
+                index = lowercaseUrn.IndexOf(':', index + 1);
```

Comment:
Just a minor thing, but you could use `originalUrn` since the method is operating with that field already